src/providers/data_providers/yelpBusinessImageProvider.ts

```diff
@@ -0,0 +1,21 @@
+import { OpenCageApiUtils } from '../../utils/openCageApiUtils';
+import { YelpFusionApiUtils } from '../../utils/yelpFusionApiUtils';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { BaseBusinessImageProvider } from './baseBusinessImageProvider';
+import { Image } from '../../models/data_models/image';
+
+export class YelpBusinessImageProvider extends BaseBusinessImageProvider {
+	protected imageProviderName: string = 'yelp';
+
+	public async getBusinessImages(businessInfo: BusinessInfoInput): Promise<Array<Image>> {
+		const yelpApi = await new YelpFusionApiUtils(this.config).init();
+		const geocode = await new OpenCageApiUtils(this.config)
```

Comment:
Eventually, this should be moved to a separate function; If we want to mix image providers, and they expect the same information (the Geocode), we don't want to make multiple of the same API request.